.codeclimate.yml

```diff
@@ -1,15 +1,167 @@
-engines:
-  golint:
+# This is our default .CodeClimate.yml, broken out by language. Uncomment the
+# sections at the bottom that apply to your project. ACTION comments indicate
+# places where config might need to be tweaked.
+
+version: "2"
+
+plugins:
+
+# ---------------
+# Cross-language plugins. Should always be on.
+
+  duplication: # Looks for similar and identical code blocks
     enabled: true
-    exclude_patterns:
-    - "**/"                # exclude all
-    - "!./cmd/"            # unexclude just the ones we want to lint
-    - "!./pkg/"
-    - "!./templates/"
+    config:
+      languages:
+        go:
+        java:
+        javascript:
+        php:
+        python:
+          python_version: 3  # ACTION Comment this out if using Python 2
+        ruby:
+        swift:
+        typescript:
 
+  fixme: # Flags any FIXME, TODO, BUG, XXX, HACK comments so they can be fixed
+    enabled: true
+    config:
+      strings:
+      - FIXME
+      - TODO
```

Comment:
@izgeri Did we reach a consensus on these?

Reply:
I updated the config to make them informational.